tests/advantage/helpers.py

```diff
@@ -49,3 +58,210 @@ def get_fixture(file: str) -> dict:
         json_data.close()
 
     return json.loads(file_data)
+
+
+def make_subscription(
+    id: str = None,
+    account_id: str = None,
+    marketplace: str = None,
+    period: str = None,
+    status: str = None,
+    last_purchase_id: str = None,
+    pending_purchases: List[str] = None,
+    items: List[SubscriptionItem] = None,
+) -> Subscription:
+    return Subscription(
+        id=id or "sAaBbCcDdEeFfGg",
```

Comment:
I'm not going to ask you to change everything, but just as a suggestion isn't it more brief to say?:
```
def make_subscription(
    id: str = "aAaBbCcDdEeFfGg",
```
And then simply
```
id=id
```
If I'm right if `id` is `None` it will always go to the `or` option.

Reply:
When I wrote it, it went through my mind to do something like this. 

I did not want to have the hard-coded values set as default inside the function definition. So I decided to go for this option instead. Either version works, yours would probably be more optimal, as you don't need the `or` statements.

Reply:
It does work, but aren't you doing the same thing with more code?